app/src/main/java/nodomain/freeyourgadget/gadgetbridge/adapter/AppBlacklistAdapter.java

```diff
@@ -126,6 +131,19 @@ public void onClick(View v) {
             }
         });
 
+        holder.btnConfigureApp.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+
+                if (holder.blacklist_checkbox.isChecked()) {
+                    Toast.makeText(mContext, R.string.toast_app_must_not_be_blacklisted, Toast.LENGTH_SHORT).show();
```

Comment:
For Toasts we have `GB.toast()`, which additionally logs them for easier understanding of bug reports.